rclpy/package.xml

```diff
@@ -38,6 +38,7 @@
   <exec_depend>action_msgs</exec_depend>
   <exec_depend>ament_index_python</exec_depend>
   <exec_depend>builtin_interfaces</exec_depend>
+  <exec_depend>python3-typing-extensions</exec_depend>
```

Comment:
This one seems like a problem as an rclpy dependency because it doesn't exist on RHEL , and [RHEL is Tier 2](https://ros.org/reps/rep-2000.html#support-tiers) which means CI is expected to pass. Is there something else we can use?

https://index.ros.org/d/python3-typing-extensions/

Reply:
Hm that's strange. `typing-extensions` is a dependency of mypy which is a dependency of ament_mypy which is built by the ROS build farm.

Reply:
Hmm in rosdep it exist for rhel. https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L9954

Reply:
Ah you're correct it does exist for RHEL https://github.com/ros/rosdistro/blob/d7c80640808f5b4ddaa80f7424c69239c84d747f/rosdep/python.yaml#L10128